modules/fedimint-lnv2-server/src/db.rs

```diff
@@ -72,7 +73,25 @@ impl_db_lookup!(
 );
 
 #[derive(Debug, Clone, Eq, PartialEq, Hash, Deserialize, Serialize, Encodable, Decodable)]
-pub struct OutgoingContractKey(pub ContractId);
+pub struct IncomingContractOutpointKey(pub ContractId);
```

Comment:
I'm a bit confused by the naming. Why is this one not indexed by `OutPoint`?

Reply:
ah ok think I answered my own question, looks like its an index from `ContractId` -> `OutPoint` -> `Contract`